android/iden3apidemo/build.gradle

```diff
@@ -8,13 +8,14 @@ android {
     ndkVersion "20.0.5594570"
 
     defaultConfig {
-        applicationId "com.iden3.gointegration"
+        applicationId "com.iden3.iden3apidemo"
```

Comment:
The purpose of this app is to check the integration of the generated Go code, I think that `idene3apidemo` is a missleading name

Reply:
this module will be used for UI testing of the iden3coreapi library once we move the tests in this module to the iden3coreapi module not needing to use the emulator or device avoiding the loadlibrary of gojni, I was checking on this and I need to finish it, but the final purpose of this module should be testing the code of the iden3mobile integration